### ISSUE TITLE
starting project on android device automatically disables lock-orientation 

### ISSUE THREAD

**Issue by abhikatta** [facebook/react-native#39361]
### Description

Whenever the project is started on a physical device the lock-orientation / auto-rotate screen is set to free even though the orientation is set to portrait in `app.json` as:
`
"orientation": "portrait",
`

### React Native Version

0.72.4

### Output of `npx react-native info`

System:
  OS: Windows 11 10.0.22621
  CPU: (8) x64 Intel(R) Core(TM) i5-9300H CPU @ 2.40GHz
  Memory: 6.3 GB / 7.85 GB
Binaries:
  Node:
    version: 18.17.1
    path: C:\Program Files\nodejs\node.EXE
  Yarn:
    version: 1.22.19
    path: ~\AppData\Roaming\npm\yarn.CMD
  npm:
    version: 9.8.1
    path: C:\Program Files\nodejs\npm.CMD
  Watchman: Not Found
SDKs:
  Android SDK: Not Found
  Windows SDK:
    AllowDevelopmentWithoutDevLicense: Enabled
IDEs:
  Android Studio: AI-223.8836.35.2231.10671973
  Visual Studio:
    - 17.7.34024.191 (Visual Studio Community 2022)
Languages:
  Java: Not Found
  Ruby: Not Found
npmPackages:
  "@react-native-community/cli": Not Found
  react:
    installed: 18.2.0
    wanted: 18.2.0
  react-native:
    installed: 0.72.4
    wanted: 0.72.4
  react-native-windows: Not Found
npmGlobalPackages:
  "*react-native*": Not Found
Android:
  hermesEnabled: Not found
  newArchEnabled: Not found
iOS:
  hermesEnabled: Not found
  newArchEnabled: Not found

### Steps to reproduce

Basically, starting the app on a physical android device does it using 
```
npm start
```
 or
```
npx expo start
```

### Snack, screenshot, or link to a repository

https://github.com/abhikatta/puwifi

**Comment by cortinico** [comment]
> is set to portrait in `app.json` as:
> `"orientation": "portrait",`

This is a Expo specific file so you should report this issue to them
